src/analysis/special_functions/arsinh.lean

```diff
@@ -0,0 +1,190 @@
+/-
+Copyright (c) 2020 James Arthur. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: James Arthur, Chris Hughes, Shing Tak Lam.
+-/
+import analysis.special_functions.trigonometric
+import data.real.basic
+import analysis.special_functions.pow
+
+open real
+local attribute [pp_nodot] real.log
+noncomputable theory
+
+/-!
+# Inverse of the sinh function
+
+In this file we prove that sinh is bijective and hence has an
+inverse, arsinh.
+
+## Main Results
+
+- `sinm_injective`: The proof that `sinm` is injective
+- `sinm_surjective`: The proof that `sinm` is surjective
+- `sinm_bijective`: The proof `sinm` is bijective
+
+## Notation
+
+- `arsinh`: The inverse function of `sinm`
```

Comment:
This is not notation, it's a definition. Notation means stuff introduced using the `notation` keyword.
I think that you should just put it under `## Main results`.

Reply:
OK, I will do. Thanks!